addons/website/views/snippets/s_masonry_block.xml

```diff
@@ -3,257 +3,234 @@
-    <section class="s_masonry_block" data-vcss="001" data-vxml="001">
-        <div class="container-fluid">
+    <section class="s_masonry_block pt48 pb48" data-vcss="001" data-vxml="001">
+        <div class="container">
             <t t-call="website.s_masonry_block_default_template"/>
         </div>
     </section>
 </template>
 
 <template id="s_masonry_block_default_template" groups="base.group_user">
-    <div class="row o_grid_mode" data-row-count="10">
-        <div class="o_grid_item col-lg-6 g-col-lg-6 g-height-10 o_grid_item_image text-center" data-name="Block"
-            style="grid-area: 1 / 1 / 11 / 7; z-index: 1;">
+    <div class="row o_grid_mode" data-row-count="8" style="gap: 16px;">
+        <div class="o_grid_item o_grid_item_image g-height-8 g-col-lg-5 col-lg-5 text-center rounded-4" data-name="Block" style="order: 0; z-index: 1; grid-area: 1 / 1 / 9 / 6;">
             <img src="/web/image/website.s_masonry_block_default_image_1" class="img img-fluid mx-auto" alt=""/>
         </div>
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-5 text-center o_cc o_cc2" data-name="Block"
-            style="grid-area: 1 / 7 / 6 / 10; z-index: 2;" >
-            <h3>A great title</h3>
-            <p>And a great subtitle</p>
+        <div class="o_grid_item o_cc o_cc2 g-height-4 g-col-lg-3 col-lg-3 justify-content-end order-lg-0 rounded-4" data-name="Block" style="order: 2; --grid-item-padding-y: 20px; --grid-item-padding-x: 20px; z-index: 2; grid-area: 1 / 6 / 5 / 9;">
+            <h3>Key <br class="d-none d-lg-inline"/>Milestone</h3>
+            <p>Reaching new heights</p>
         </div>
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-5 text-center o_cc o_cc3" data-name="Block"
-            style="grid-area: 1 / 10 / 6 / 13; z-index: 3;">
-            <h3>A great title</h3>
-            <p>And a great subtitle</p>
+        <div class="o_grid_item o_cc o_cc3 g-height-4 g-col-lg-4 col-lg-4 justify-content-end order-lg-0 rounded-4" data-name="Block" style="order: 1; --grid-item-padding-y: 20px; --grid-item-padding-x: 20px; z-index: 3; grid-area: 1 / 9 / 5 / 13; background-image: linear-gradient(135deg, var(--o-color-4) -400%, var(--o-color-2) 100%);">
+            <h3>Greater <br class="d-none d-lg-inline"/>Impact</h3>
+            <p>Making a difference every day</p>
         </div>
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-5 text-center o_cc o_cc4" data-name="Block"
-            style="grid-area: 6 / 7 / 11 / 10; z-index: 4;">
-            <h3>A great title</h3>
-            <p>And a great subtitle</p>
+        <div class="o_grid_item o_cc o_cc4 g-height-4 g-col-lg-4 col-lg-4 justify-content-end order-lg-0 rounded-4" data-name="Block" style="order: 2; --grid-item-padding-y: 20px; --grid-item-padding-x: 20px; z-index: 4; grid-area: 5 / 6 / 9 / 10; background-image: linear-gradient(135deg, var(--o-color-4) -400%, var(--o-color-1) 100%);">
```

Comment:
It's intentional actually. The actual order between two elements with the same `order` value is defined by the DOM and I prefer to be conservative by tweaking the order only when necessary (I don't know how far make sense to go since, for example, BS has "only" 5 order levels).
We can change it tho, let me know :)

Reply:
Given that moving the elements on mobile depends on them, it should be set as the move option expects it so they should be incremental (so like I said) :slightly_smiling_face: (It is not simply like with bootstrap, when we move in mobile for the first time, it sets one order per element, so none of them have the same order :wink:)   

If you go in mobile view, you notice that the arrows to move the elements are inconsistent, because they depend on the orders (here, since the biggest order is 2, it doesn't expect a 4th and 5th element, so for the arrows, there is no next element)  :arrow_down: 
![image](https://github.com/user-attachments/assets/0234207d-44dc-4af5-85d9-96a178214b3e)

Reply:
I see, thanks for the explanation 👍